java/BinExport/src/main/java/com/google/security/binexport/BinExportExporter.java

```diff
@@ -85,7 +85,10 @@ public boolean export(File file, DomainObject domainObj,
       final BinExport2 proto = builder.build(monitor);
 
       monitor.setMessage("Writing BinExport2 file");
-      proto.writeTo(new FileOutputStream(file));
+
+      var outputStream = new FileOutputStream(file);
```

Comment:
This should use its own try-with-resources block then, no? Otherwise, if an exception happens in `proto.writeTo()` we end up in the same situation.
Please also mark local variables as `final` if possible.

Reply:
Wow, I didn't know these existed. Thanks for the hint. It's been a while since I programmed in java. I'll fix it